### ISSUE TITLE
[BUG Report] llama int8 PPL drop a lot with no cublaslt support.

### ISSUE THREAD

**Issue by MeJerry215** [TimDettmers/bitsandbytes#672]
if has cublas lt support,  load_in_8bit accuracy is ok.
but if compiled a version without cublaslt support or compute capability less then 7.5, it will use torch.nn.functional.linear to calculate the value.
And I found that, in this case, llama int8 PPL drop a lot from 7.5 to 18. I don't know how this happend. 
and I just replace the half calculate with quantized value the PPL recover.

![image](https://github.com/TimDettmers/bitsandbytes/assets/53092165/1d0a043e-c876-4794-8b14-0fafec12f696)
